unconvert.go

```diff
@@ -425,7 +427,13 @@ func (v *visitor) unconvert(call *ast.CallExpr) {
 		return
 	}
 
-	v.edits[v.file.Position(call.Lparen)] = struct{}{}
+	pos := v.file.Position(call.Lparen)
+	if !filepath.IsAbs(pos.Filename) { // go 1.11+, see https://github.com/golang/go/issues/26671
```

Comment:
For go<1.11 filename will be `a/b.go`, but in go1.11 filename will be `b.go`. Absolute path checking doesn’t help.
IMO it should be fixed in go/scanner: it’s an incompatible change and we need to fix dozens of tools because of it

Reply:
> For go<1.11 filename will be a/b.go

I don't think so. See https://github.com/golang/go/issues/26671, the code (removed by https://github.com/golang/go/commit/546bab8c295895368ce8ec3d97ff65472c1032ab) was:
```go
	if filename != "" {
￼ 		filename = filepath.Clean(filename)
￼ 		if !filepath.IsAbs(filename) {
￼ 			// make filename relative to current directory
￼ 			filename = filepath.Join(s.dir, filename)
￼```

Here  `s.dir` is the path from the original file name supplied to scanner.Init(), and it could either be relative or absolute. Due to a way unconvert handles input (using golang.org/x/tools/go/loader) file names supplied are absolute.

The above it theory; my tests confirm it.